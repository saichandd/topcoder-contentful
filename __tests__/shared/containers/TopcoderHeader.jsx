import _ from 'lodash';
import Container from 'containers/TopcoderHeader';
import React from 'react';
import R from 'react-test-renderer/shallow';

const mockState = {
  auth: {
    profile: {},
  },
  topcoderHeader: {},
  currentNav: {
    menuTitle: 'title',
    subMenuTitle: 'subtitle',
  },
};

test('Matches shallow snapshot', () => {
  const r = new R();
  r.render((
    <Container
      store={{
        dispatch: () => _.noop,
        getState: () => mockState,
        subscribe: _.noop,
      }}
    />
  ));
  expect(r.getRenderOutput()).toMatchSnapshot();
});
