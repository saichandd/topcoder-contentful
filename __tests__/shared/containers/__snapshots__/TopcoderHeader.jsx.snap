// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow snapshot 1`] = `
<TopcoderHeader
  activeTrigger={null}
  closeMenu={[Function]}
  closeMobileMenu={[Function]}
  closeSearch={[Function]}
  isMobile={false}
  mobileMenuOpened={false}
  openMenu={[Function]}
  openMobileMenu={[Function]}
  openSearch={[Function]}
  openedMenu={null}
  profile={Object {}}
  searchOpened={false}
  setCurrentNav={[Function]}
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "subscribe": [Function],
    }
  }
  storeSubscription={
    Subscription {
      "listeners": Object {
        "notify": [Function],
      },
      "onStateChange": [Function],
      "parentSub": undefined,
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "subscribe": [Function],
      },
      "unsubscribe": null,
    }
  }
/>
`;
